src/member/signals/handlers.py

```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+from ..models import ClientScheduledStatus
+
+
+@receiver(post_delete, sender=ClientScheduledStatus)
+def ensure_pair_remove(sender, instance, **kwargs):
+    if instance.get_pair:
+        ClientScheduledStatus.objects.get(pk=instance.get_pair.pk).delete()
```

Comment:
@lingxiaoyang and @manumilou, Ling has been made an interesting questions.

I think that is better avoid to remove the status if that status was processed, we can keep the status as `Operation Status: Processed`, because is totally true, and avoid to show the button `Remove` for the processed statuses.